AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -111,16 +109,17 @@ struct _ABMeasurementsView: View {
     private var streamNames: some View {
         return HStack {
             Group {
+                streamsToShow.count != 1 ? Spacer() : nil
                 ForEach(streamsToShow, id : \.self) { stream in
                     SingleMeasurementView(stream: stream,
                                           threshold: nil,
                                           selectedStream: $selectedStream,
                                           isCollapsed: $isCollapsed,
                                           measurementPresentationStyle: .hideValues,
                                           isDormant: session.isDormant)
+                    Spacer()
```

Comment:
I don't think using a Spacer() here is a good idea, right now the AB parameters are squeezed in the middle and there's too much space on the sides. Maybe try to use horizontal `.padding()` instead? :)

Reply:
Here is just an example, don't hesitate to do it another way:

```
private var streamNames: some View {
        return HStack {
            Group {
                ForEach(streamsToShow, id : \.self) { stream in
                    SingleMeasurementView(stream: stream,
                                          threshold: nil,
                                          selectedStream: $selectedStream,
                                          isCollapsed: $isCollapsed,
                                          measurementPresentationStyle: .hideValues,
                                          isDormant: session.isDormant)
                    streamsToShow.count == 1 ? Spacer() : nil
                }
            }
            .padding(.horizontal)
        }
    }
```